docs/src/faq.md

```diff
@@ -88,3 +90,250 @@ Where can I find the nitty-gritty details?
+
+Inscriptions for…
+-----------------
+
+### Artists
+
+*Inscriptions are on Bitcoin.* Bitcoin is the digital currency with the highest
+status and greatest chance of long-term survival. If you want to guarantee that
+your art survives into the future, there is no better way to publish it than as
+inscriptions.
+
+*Cheaper on-chain storage.* At $20,000 per BTC and the minimum relay fee of 1
+satoshi per vbyte, publishing inscription content costs $50 per 1 million
+bytes.
+
+*Inscriptions are early!* Inscriptions are still in development, and have not
+yet launched on mainnet. This gives you an opportunity to be an early adopter,
+and explore the medium as it evolves.
+
+*Inscriptions are simple.* Inscriptions do not require writing or understanding
+smart contracts.
+
+*Inscriptions unlock new liquidity.* Inscriptions are more accessible and
+appealing to bitcoin holders, unlocking an entirely new class of collector.
+
+*Inscriptions are designed for digital artifacts.* Inscriptions are designed
+from the ground up to support NFTs, and feature a better data model, and
+features like globally unique symbols and enhanced provenance.
+
+*Inscriptions do not support on-chain royalties.* This is negative, but only
+depending on how you look at it. On-chain royalties have been a boon for
+creators, but have also created a huge amount of confusion in the Ethereum NFT
+ecosystem. The ecosystem now grapples with this issue, and is is engaged in a
+race to the bottom, towards a royalties-optional future. Inscriptions have no
+support for on-chain royalties, because they are technically infeasible. If you
+choose to create inscriptions, there are many ways you can work around this
+limitation: withhold a portion of your inscriptions for future sale, to benefit
+from future appreciation, or perhaps offer perks for users who respect optional
+royalties.
+
+### Collectors
+
+*Inscriptions are simple, clear, and have no surprises.* They are always
+immutable and on-chain, with no special due diligence required.
+
+*Inscriptions are on Bitcoin.* You can verify the location and properties of
+inscriptions easily with Bitcoin full node that you control.
+
+### Bitcoiners
+
+Let me begin this section by saying: the most important thing that the bitcoin
+network does is decentralize money. All other use-cases are secondary,
+including ordinal theory. The developers of ordinal theory understand and
+acknowledge this, and believe that ordinal theory helps, at least in a small
+way, Bitcoin's primary mission.
+
+Unlike many other things in the altcoin space, digital artifacts have merit.
+There are, of course, a great deal of NFTs that are ugly, stupid, and
+fraudulent. However, there are many that are fantastically creative, and
+creating and collecting art has been a part of the human story since its
+inception, and predates even trade and money, which are also ancient
+technologies.
+
+Bitcoin provides an amazing platform for creating and collecting digital
```

Comment:
Maybe also mention that it encourages more people to run a full bitcoin node -> more decentralization.